ts/test/automation/setup/beforeEach.ts

```diff
@@ -7,40 +7,46 @@ import { MULTI_PREFIX, NODE_ENV, openElectronAppOnly } from './open';
 const getDirectoriesOfSessionDataPath = (source: string) =>
   readdirSync(source, { withFileTypes: true })
     .filter(dirent => dirent.isDirectory())
-    .map(dirent => dirent.name)
-    .filter(n => n.startsWith(`Session-${NODE_ENV}-${MULTI_PREFIX}`));
+    .map(dirent => {
```

Comment:
Why did it need you to expand the function?

Reply:
I just added a debug log and thought it would be nice to keep it that way if we ever need to debug it again:)

Reply:
Fair enough